src/page/slot.rs

```diff
@@ -266,21 +214,25 @@ impl<T, C: cfg::Config> Slot<T, C> {
         // will have to be removed when those references are dropped.
         if refs.value > 0 {
             return true;
```

Comment:
nit/style: since this `if` statement is now the last expression in the method, we shouldn't need the `return` any longer:
```suggestion
            true
```